tests/integration_tests/test_hyperopt.py

```diff
@@ -314,3 +315,96 @@ def test_hyperopt_run_hyperopt(csv_filename, search_space, tmpdir, ray_cluster):
 
     if os.path.isfile(os.path.join(tmpdir, "test_hyperopt", "hyperopt_statistics.json")):
         os.remove(os.path.join(tmpdir, "test_hyperopt", "hyperopt_statistics.json"))
+
+
+@pytest.mark.distributed
+@pytest.mark.parametrize("search_space", ["random"])
+def test_hyperopt_run_hyperopt_with_shared_params(csv_filename, search_space, tmpdir, ray_cluster):
+    input_features = [
```

Comment:
You can abstract the initialization of the Ludwig config into a separate function, see `test_hyperopt_search_alg` above. This was you can write follow up tests on the same config if needed.

You can either edit the existing `_setup_ludwig_config` function or add a new ludwig config setup method that is specific for your test.